.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 *       @BrightspaceUI/gaudi-dev
+package-lock.json
```

Comment:
This is maybe a dumb question but what does it mean to have `package-lock.json` in the codeowners file?

Reply:
It's essentially saying that if a change is happening only to the lock file (which is what this `update-package-lock` workflow does), then there are no CODEOWNERS and our requirement that PRs are approved by a CODEOWNER goes away, enabling auto-merge.